Building outside of workdir fails

I'm using branch v0.7.x of anlsys/libnrm@f999916 and anlsys/nrm-benchmarks@cbf1162

I'm assuming libnrm is correctly installed under `/tmp/nrm/install`.

When I try to build in `/tmp/nrm/build/nrm-benchmarks`, the compilation does not complete as it fails to find the header `nrm-benchmarks.h`.

Sequence of commands (output removed, assuming bash) to reproduce the issue:
```console
$ mkdir -p /tmp/nrm/{build,install,src}
$ cd /tmp/nrm/src
$ git clone git@github.com:anlsys/nrm-benchmarks.git
$ cd nrm-benchmarks
$ git switch v0.7.x
$ ./autogen.sh
$ cd /tmp/nrm/build/nrm-benchmarks/
$ /tmp/nrm/src/nrm-benchmarks/configure --prefix=/tmp/nrm/install
$ make
depbase=`echo src/utils.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
gcc -DHAVE_CONFIG_H -I. -I/tmp/nrm/src/nrm-benchmarks -I./src  -Isrc -fopenmp -I/tmp/nrm/install/include -isystem /usr/include/mit-krb5 -I/usr/include/pgm-5.3 -I/usr/include/libxml2    -g -O2 -MT src/utils.o -MD -MP -MF $depbase.Tpo -c -o src/utils.o /tmp/nrm/src/nrm-benchmarks/src/utils.c &&\
mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/progress/ones/stream/copy.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
gcc -DHAVE_CONFIG_H -I. -I/tmp/nrm/src/nrm-benchmarks -I./src  -Isrc -fopenmp -I/tmp/nrm/install/include -isystem /usr/include/mit-krb5 -I/usr/include/pgm-5.3 -I/usr/include/libxml2    -g -O2 -MT src/progress/ones/stream/copy.o -MD -MP -MF $depbase.Tpo -c -o src/progress/ones/stream/copy.o /tmp/nrm/src/nrm-benchmarks/src/progress/ones/stream/copy.c &&\
mv -f $depbase.Tpo $depbase.Po
/tmp/nrm/src/nrm-benchmarks/src/progress/ones/stream/copy.c:13:10: fatal error: nrm-benchmarks.h: No such file or directory
   13 | #include "nrm-benchmarks.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:607: src/progress/ones/stream/copy.o] Error 1
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@
 ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST = autogen.sh README.markdown
 
-AM_CPPFLAGS = -Isrc $(OPENMP_CFLAGS) $(LIBNRM_CFLAGS)
+AM_CPPFLAGS = -I$(top_srcdir)/src $(OPENMP_CFLAGS) $(LIBNRM_CFLAGS)
 AM_LDFLAGS = $(OPENMP_CFLAGS) $(LIBNRM_LIBS)
 
 include_HEADERS=src/nrm-benchmarks.h

